src/Behat/Config/Converter/UsedClassesCollector.php

```diff
@@ -14,9 +14,28 @@ class UsedClassesCollector extends NodeVisitorAbstract
 
     public function enterNode(Node $node): ?Node
     {
-        if (!$node instanceof Node\Name\FullyQualified) {
-            return null;
+        if ($node instanceof Node\Name\FullyQualified) {
+            return $this->importClass($node);
         }
+
+        if ($node instanceof Node\Expr\ClassConstFetch && $node->class instanceof Node\Name) {
+            // E.g. a `MyClass::class` argument
+            return new Node\Expr\ClassConstFetch(
+                $this->importClass($node->class),
+                $node->name,
+                $node->getAttributes(),
+            );
+        }
+
+        if ($node instanceof Node\Expr\ClassConstFetch) {
+            var_dump($node->class::class);
```

Comment:
I need to take a deeper look but in the meantime you probably want to remove this 😉

Reply:
Indeed I do 🤦‍♂️😆

Reply:
Fixed, thanks :)